### PR TITLE
doc/developer_guide: add `Email notifications` section

### PR DESCRIPTION
**Pull request by JenySadadia**
Explain KCIDB email notifications.

### CODE REVIEW

**Comment by JenySadadia** [comment]
@spbnick Is it the correct place to mention email notifications or should I add this section to the submitter guide?

**Comment by JenySadadia** [comment]
OK, I'll add a new page.

File: doc/subscriber_guide.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+---
+Email notifications
+---------------------
+
+It is possible to generate email notifications with the KCIDB data submissions. It needs to have a [subscription module][subscriptions], a script that will create a custom report using [templates](https://github.com/kernelci/kcidb/tree/main/kcidb/templates). A list of subscribers would receive the report via an email notification.
+
+Currently, multiple subscriptions are active for receiving email reports including `stable-rc` and `stable-rt` reports.
+The reports list useful data such as build and test failures with architecture and build config information reported by different CI systems.
+
+Here is an example of such report:
```
Comment (spbnick) [inline doc/subscriber_guide.md:16]: Could you wrap the paragraphs on 78 columns?

File: doc/subscriber_guide.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    https://kcidb.kernelci.org/d/revision/revision?orgId=1&var-git_commit_hash=b57d01c66f40ec96d8a5df6c2cdbd75e15e5f07c&var-patchset_hash=
+
+
+Tested-by: kernelci.org bot <bot@kernelci.org>
+
+Thanks,
+KernelCI team
+```
+
+Such reports are useful for developers and maintainers to keep track of new failures with each new revision and can use these data for further required actions.
```
Comment (spbnick) [inline doc/subscriber_guide.md:77]: And here too, please.

**Review by spbnick — CHANGES_REQUESTED** [review]
Thank you, Jeny! This looks good. I would only like to see better formatting of the Markdown source, where possible, as its whole point (ok, maybe not the whole, but most) is to be readable as is, and long lines are not that readable.

And another nitpick: since the subscriber guide did not exist before your commit, the correct commit subject would look something like this: `doc: Add a submitter guide`.

**Review by spbnick — APPROVED** [review]
LGTM. Thank you, Jeny!

**Pull request merged**